scripts/declarations/utils/index.js

```diff
@@ -33,48 +36,55 @@ export default class DeclarationUtils {
     return modifiedServicesIds;
   }
 
-  async getModifiedServiceTermsTypes() {
-    const { modifiedFilePaths, modifiedServicesIds } = await this.getModifiedData();
+  async getModifiedServicesTermsTypes() {
+    const { modifiedFilePaths } = await this.getModifiedData();
     const servicesTermsTypes = {};
 
     await Promise.all(modifiedFilePaths.map(async modifiedFilePath => {
-      const serviceId = DeclarationUtils.filePathToServiceId(modifiedFilePath);
+      const serviceId = DeclarationUtils.getServiceIdFromFilePath(modifiedFilePath);
+
+      if (modifiedFilePath.endsWith('.filters.js')) {
+        const declaration = await this.getJSONFromFile(this.defaultBranch, `declarations/${serviceId}.json`);
 
-      if (!modifiedFilePath.endsWith('.json')) {
-        // Here we should compare AST of both files to detect on which function
-        // change has been made, and then find which terms type depends on this
-        // function.
-        // As this is a complicated process, we will just send back all terms types
-        const declaration = await this.getJSONFile(`declarations/${serviceId}.json`, this.defaultBranch);
+        // TODO: Implement AST comparison between original and modified "filters" files to detect changes in functions, enabling identification of terms whose types depend on modified functions.
+        // Due to the complexity involved, temporarily returning all term types.
```

Comment:
Good rephrasing of the comment!
I am not sure of the relevance of the content however, no matter the form. Indeed:

1. This is as much a TODO as any other improvement in the codebase. We typically don't use this TODO / FIXME syntax in the code to keep track of potential improvements but use project tracking tools.
2. I am not sure we need “AST comparison”. Every JS function can be `toString`-ed and one possible implementation could thus be to compare the stringified content of every `filter` entry in the declaration and compare it to the previous version.

Since this comment seems to be “programmer’s side notes” rather than providing a relevant explanation for a design choice at line level, I am in favour of rephrasing further.
```suggestion
         // Considering how rarely filters are used, simply return all term types that could potentially be impacted to spare implementing a function change check
```